src/account_multisig.ts

```diff
@@ -314,7 +314,54 @@ export class Account2FA extends AccountMultisig {
         }
     }
 
-    async disable(contractBytes: Uint8Array, cleanupContractBytes: Uint8Array) {
+    async disableWithFAK({ contractBytes, cleanupContractBytes }: { contractBytes: Uint8Array, cleanupContractBytes?: Uint8Array }) {
+        let cleanupActions = [];
+        const keyConversionActions = await this.get2faDisableKeyConversionActions();
+        if(cleanupContractBytes) {
+            let deleteAllRequestsError;
+            await this.deleteAllRequests().catch(e => deleteAllRequestsError = e);
+            cleanupActions = await this.get2faDisableCleanupActions(cleanupContractBytes).catch(e => {
+                if(e.type === 'ContractHasExistingState') {
+                    throw deleteAllRequestsError || e;
+                }
+                throw e;
+            });
+        }
+
+        const actions = [
+            ...cleanupActions,
+            ...keyConversionActions,
+            deployContract(contractBytes)
+        ];
+
+        const accessKeyInfo = await this.findAccessKey(this.accountId, actions);
+
+        if(accessKeyInfo?.accessKey?.permission !== 'FullAccess') {
+            throw new TypedError(`No full access key found in keystore. Unable to bypass multisig`, 'NoFAKFound');
+        }
+
+        return this.signAndSendTransactionWithAccount(this.accountId, actions);
+    }
+
+    async get2faDisableCleanupActions(cleanupContractBytes: Uint8Array) {
+        const currentAccountState: { key: Buffer, value: Buffer }[] = await this.viewState('').catch(error => {
+            const cause = error.cause && error.cause.name;
+            if (cause == 'NO_CONTRACT_CODE') {
+                return [];
+            }
+            throw cause == 'TOO_LARGE_CONTRACT_STATE' ?
+                new TypedError(`Can not deploy a contract to account ${this.accountId} on network ${this.connection.networkId}, the account has existing state.`, 'ContractHasExistingState') :
+                error;
```

Comment:
Suggestion for a bit easier reading:

```suggestion
            throw cause == 'TOO_LARGE_CONTRACT_STATE'
                ? new TypedError(`Can not deploy a contract to account ${this.accountId} on network ${this.connection.networkId}, the account has existing state.`, 'ContractHasExistingState')
                : error;
```